shared/naturalcrit/markdown.js

```diff
@@ -238,7 +268,38 @@ const definitionLists = {
 	}
 };
 
-Marked.use({ extensions: [mustacheSpans, mustacheDivs, mustacheInjectInline, definitionLists] });
+let pageBlockTopLevel = true;
+let pageBlockNumber = 0;
+const pageBlocks = {
+	name  : 'pageBlock',
+	level : 'block',
+	start(src) { return pageBlockTopLevel; },
+	tokenizer(src, tokens) {
+		if(!pageBlockTopLevel) return false;
+		const pageArray = src.split(/^\\page$/gm);
+		if(this.lexer.tokens.length == 0 && this.lexer.state.top) pageBlockNumber = 0;
+		pageBlockNumber++;
+
+		const token = {
+			type       : 'pageBlock',
+			raw        : `${pageArray[0]}\\page`,
```

Comment:
This can probably be checked by looking if the first split is an empty string "".  If so, we know the current string starts with `\page` (in which case we grab the second split).

Reply:
The latest commit should have improved this.